hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java

```diff
@@ -370,6 +371,7 @@ public Object getProperty(final String name) {
 			entry("fees.tokenTransferUsageMultiplier", AS_INT),
 			entry("fees.percentCongestionMultipliers", AS_CONGESTION_MULTIPLIERS),
 			entry("files.maxSizeKb", AS_INT),
+			entry("ledger.id", AS_STRING),
```

Comment:
`AS_STRING` is [the default transform](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java#L300), so we don't need to repeat it here.

Reply:
removed the entry. Thanks !